### ISSUE TITLE
Question:on deployment : users of this project which hosting provider you chose?

### ISSUE THREAD

**Issue by rajesh188** [bernardopires/django-tenant-schemas#584]
Because after buying a 12$ hacker account on python anywhere.com I read in its forum that it does not yet support multi domain inside app i.e *.myapp.pythonanywhere.com.
My question is I have built a robust django project with tenant_schemas and need help with deploying it as is.i am able to run it locally with no issues.i have zero experience with hosting on server/DNS.
 
Pls help/guide/point me in right direction to deploy my django project with this awesome lib

**Comment by rj76** [comment]
What exactly is your question? I have some experience in deploying a multi tenant setup with django-tenant-schemas.

**Comment by rajesh188** [comment]
Which hosting provider allows multiple subdomains ? I tried pythonanywhere and unfortunately it doesn't allow subdomains like tenant.myproject.pythonanywhere.com.

**Comment by rj76** [comment]
I manage my domains (and thus it's subdomains) with dynadot, and host my servers at Linode. So it's not a question of your hosting provider; if you can manage your DNS, just point it to the server you want you app to run and catch it with, for example, nginx.

**Comment by rajesh188** [comment]
Basically I have solved a business problem and implemented it in django since it is useful for many I want to serve same solution to more customers hence chose this project but I'm stuck at choosing hosting provider who will allow me to serve different customers on subdomain .

 Tldr : does heroku good choice to deploy this project?

**Comment by rajesh188** [comment]
I don't own a domain currently.im thinking of using heroku free tier first.then scale up

**Comment by rj76** [comment]
I don't know anything about Heroku, but if you can catch all subdomain trafic and point it to yoour django app it should be fine.

**Comment by rajesh188** [comment]
Maybe I should read up on domains and hosting.thank you for your valuable insight

**Comment by rajesh188** [comment]
Ok I chose Google cloud
